.github/workflows/test.yml

```diff
@@ -21,14 +21,15 @@ jobs:
   workspace_deluxe_tests:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Most important question - have you fired up the server in a docker container and made sure it works? I think minimum config would be a local mongo and pointing it at CI auth

Reply:
Can we come up with a minimal test (or test set) that could be run on the built image to ensure that it is functional? That would be better, so that there's some sort of framework in place for the next migration (in twenty years' time).

Reply:
yeah, it's been in my mental to do list for a long time. I was thinking along the lines of a script that runs docker compose to start mongo, auth in test mode (if that's possible with the current auth docker image, not sure), and the ws, creates a user & token in auth, and then creates and reads a workspace to make sure the service is working.

Reply:
Later some of the other services could be added (blobstore, handle service, sample service), but I think the tests cover the interactions there pretty well

Reply:
perhaps also fetching a page from the doc server?

Reply:
Sure, although the only thing that could go wrong there is a messed up web.xml file. The tests show the internals work and the hypothetical docker test would show the tomcat stuff works, so really the only remaining part of the picture is mounting the jar. Wouldn't hurt though

Reply:
All that being said, currently there just needs to be, at minimum, some manual test of the server for this PR

Reply:
See below for minimal auth2 / workspace integration-type tests.

Reply:
Nice! :+1: